packages/client/src/pages/common/nav/nav.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { AppBar, Toolbar, Typography, Button,ButtonBase } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+
+const useStyles = makeStyles((theme: any) => ({
```

Comment:
Are we using styledcomponents or mui? MUI prob cleaner.

Reply:
Since most of this component is mui, I went with mui... probably ok if consistency with other components doesn't matter

Reply:
I think `styled-components` overrides any mui styles so its fine to use with MUI components (I set this up). Just wondering this for consistency's sake.

Reply:
Yeah I'll convert this to styled-components in my next navbar change, but gonna merge this for tomorrow since I can't work on it tonight

Reply:
Leave these as unresolved and I"ll come back to it xd